_data/projects/Botwin.yml

```diff
@@ -0,0 +1,16 @@
+name: Botwin
+desc: Botwin is a library that allows Nancy-esque routing for use with ASP.Net Core.
+site: *https://github.com/jchannon/Botwin
```

Comment:
Uh, I don't think this `*` needs to be here. But I don't think that's the error here 🤔